ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -804,7 +815,7 @@ else
                 // Resize iframe so the frame has no scrollbars
                 // By the time the window has loaded, the iframe contents should have as well, thus no iframe load listener
                 // Note that when the puzzle receives the message, it should remove any extraneous wrapping before reporting its size
-                const iframe = document.querySelector("iframe");
+                const iframe = document.getElementById("puzzle-frame");
```

Comment:
While you're making edits, querySelector is the preferred method for selection, so you could just change the `iframe` to `#puzzle-frame`.